analytics_dashboard/templates/footer.html

```diff
@@ -3,11 +3,26 @@
 
 {#  Translators: Application here refers to the web site/application being used (e.g. the dashboard). #}
 <footer class="footer" aria-label="{% trans "Application" %}">
-  {% if show_feedback %}
-  <div class="feedback">
-    <div class="container">
-      <div class="row">
 
+  <div class="container">
+    <div class="row colophon">
+      <nav class="nav-colophon" aria-label="${_('About')}">
+        <ol>
```

Comment:
Let's pull this list of links from settings like the LMS.  These values are pretty Credo specific, obviously.  :)